### PR TITLE
UC-203 | Fix forum boards with slashes

### PR DESCRIPTION
**Pull request by kvas-damian**

### CODE REVIEW

File: extensions/wikia/Wall/js/Wall.js [hunk 1]
```diff
@@ -19,7 +19,7 @@
 			this.deletedMessages = {};
 			this.isMonobook = window.skin && window.skin === 'monobook';
 			this.hasMiniEditor = typeof window.wgEnableMiniEditorExt !== 'undefined' && !this.isMonobook;
-			this.title = window.wgTitle.split('/')[0];
+			this.title = window.wgTitle;
```
Comment (kvas-damian) [inline extensions/wikia/Wall/js/Wall.js]: Didn't find place on MessageWall where you could get ```wgTitle``` that contains ```/```. That's why I think this is valid fix.
The only place in Wall/Forum where I was able to create page that ```wgTitle``` contains ```/``` was board and this ```.split()``` was causing issues there.

**Comment by kvas-damian** [comment]
ping @nandy-andy

**Comment by armonr** [comment]
Good find @kvas-damian 
However, this does not seem to be the problem with the forums that are created under a wrong (probably default) forum board. Probably an issue in the Forum ext.

**Comment by kvas-damian** [comment]
@armonr I expect that next sentence will drastically change your point of view on this extension ;) **Forum code is strongly dependent on Wall code** - including both frontned and backend :boom: .

Let's analyze this case:
[... missed some steps to not blur main flow]
1. magic starts here: https://github.com/Wikia/app/blob/dev/extensions/wikia/Forum/js/ForumNewMessageForm.js#L56 - as you can see this line calls ```doPostNewMessage``` on parent.
2. ok, let's find parent:  ```NewMessageForm``` extends ```Wall.settings.classBindings.newMessageForm``` - https://github.com/Wikia/app/blob/dev/extensions/wikia/Forum/js/ForumNewMessageForm.js#L7 which is defined in https://github.com/Wikia/app/blob/dev/extensions/wikia/Wall/js/WallNewMessageForm.js
3. we found real ```doPostNewMessage``` method - https://github.com/Wikia/app/blob/dev/extensions/wikia/Wall/js/WallNewMessageForm.js#L81 which uses ```this.page```  - we will get back to this later.
4. Next step is to call ```postNew``` in https://github.com/Wikia/app/blob/dev/extensions/wikia/Wall/js/WallBackendBridge.js#L40 which uses parameter that I mentioned before - https://github.com/Wikia/app/blob/dev/extensions/wikia/Wall/js/WallBackendBridge.js#L50
5. Ok everything fine, but where the hell ```this.page``` is set? If go through whole class hierarchy you'll finally find that ```this.page``` is created in https://github.com/Wikia/app/pull/6021/files#diff-f3ee51102cd27b13d69d630f527eb427R23

Hope, this will help code reviewing my change.

Oh, almost forgot. Thread creation and pagination is handled via Ajax and Board name is taken from ```this.page``` attribute, that's why this line fixes both issues.

**Comment by nandy-andy** [comment]
As we discussed: I don't recall possibility in the past to have any sub-page in `MessageWall` namespace. Therefore, there is no reason to split `wgTitle` by `/` and if this breaks forum the change LGTM :japanese_ogre:

**Comment by armonr** [comment]
Thanks for your response. When I tested this, I removed the auto-generated template in the sub-forum body before saving which caused the page to disappear from sub-forum list under the forum I expected it to appear! That is not however, related to slashes, I realize.
:+1:

**Pull request merged**